admin/class-smaily-for-wp-admin.php

```diff
@@ -34,8 +43,9 @@ class Smaily_For_WP_Admin {
 	 * @param string $version     The version of this plugin.
 	 */
 	public function __construct( $plugin_name, $version ) {
-		$this->plugin_name = $plugin_name;
-		$this->version     = $version;
+		$this->plugin_name    = $plugin_name;
+		$this->version        = $version;
+		$this->options = new Smaily_For_WP_Options();
```

Comment:
Formatting: invalid equal sign alignment.

Reply:
Fixed this.